app/workers/podcast_episodes/bust_cache_worker.rb

```diff
@@ -2,9 +2,10 @@ module PodcastEpisodes
   class BustCacheWorker < BustCacheBaseWorker
     def perform(podcast_episode_id, path, podcast_slug)
       podcast_episode = PodcastEpisode.find_by(id: podcast_episode_id)
-      return if podcast_episode.nil?
 
-      CacheBuster.bust_podcast_episode(podcast_episode, path, podcast_slug)
+      return unless podcast_episode && path && podcast_slug
```

Comment:
Optimization suggestion: since we don't care about busting a podcast episode unless both `path` and `podcast_slug` are present, could we check for those first (before querying the db for `PodcastEpisode` on [line 4](https://github.com/forem/forem/pull/12049/files#diff-1637a65634c9deae0cf148627c012306f1c54cbc2c37ff48589a3e70da93ebddR4))? Seems like we could avoid an unnecessary query, although admittedly, maybe it's not _that_ much of an optimization 🙃

Reply:
Good idea - updated!